src/routes/webhooks.ts

```diff
@@ -166,7 +166,7 @@ export default async function routes(fastify: FastifyInstance) {
         }
 
         default:
-          throw new Error('Unhandled webhook event')
+          return reply.send({ received: true, handled: false })
```

Comment:
This is a breaking change - for now, simply do not send any unsupported events to the webhook